src/test/java/net/honux/chess/core/BoardTest.java

```diff
@@ -0,0 +1,39 @@
+package net.honux.chess.core;
+
+import net.honux.chess.attribute.Color;
+import net.honux.chess.entity.pieces.Pawn;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.*;
+
+class BoardTest {
+
+    Board board;
+
+    @BeforeEach
+    void generateBoard() {
+        board = new Board();
+    }
+
+    @Test
+    @DisplayName("Pawn 이 보드에 정상적으로 등록되어야 한다.")
+    void createPawnOntheBoard() {
+        Pawn white = new Pawn(Color.WHITE);
+        Pawn black = new Pawn(Color.BLACK);
+        board.add(white);
+        Assertions.assertAll(
+                ()->assertThat(board.size()).isEqualTo(1),
+                ()->assertThat(white).isEqualTo(board.findPawn(0).get()));
+        board.add(black);
+        Assertions.assertAll(
+                ()->assertThat(board.size()).isEqualTo(2),
+                ()->assertThat(black).isEqualTo(board.findPawn(1).get()));
```

Comment:
`assertAll()` 적용 좋습니다.
하지만, 되도록 테스트는 테스트 하고자 하는 부분을 아래쪽으로 몰아주세요.
테스트 하고자 하는 바를 더 명확히 알 수 있도록 가독성이 좋아지게 됩니다.

저라면 아래와 같이 구성했을 것 같습니다.
```suggestion
    void createPawnOntheBoard() {
        Pawn white = new Pawn(Color.WHITE);
        Pawn black = new Pawn(Color.BLACK);
        
        board.add(white);
        board.add(black);
        
        assertAll(
                () -> assertThat(board.size()).isEqualTo(2),
                () -> assertThat(white).isEqualTo(board.findPawn(0).get()));
                () -> assertThat(black).isEqualTo(board.findPawn(1).get()));
```

Reply:
그렇군요 AssertAll() 을 저렇게 생각하면 좋겠군요.

커밋 메세지 관련한건 이번 연휴간 읽어보고 커밋 템플릿을 설정해놓아야겠군요

Reply:
ㅎㅎ 커밋메시지는 규칙성이 있는게 보기좋긴 합니다.